.github/workflows/docker.yaml

```diff
@@ -3,10 +3,31 @@ name: Reusable CI workflow for Docker Build & Push to ECR
 on:
   workflow_call:
     inputs:
+      name: 
+        description: "Application Name overwrite (Default is the github repo name)"
+        required: false
+        type: string
+        default: ${{ github.event.repository.name  }}
       trivy_exit_code:
+        description: "if set to 0, workflow will ignore trivy scan errors.(not recommended)"
```

Comment:
should be "if set to 0, workflow will ignore trivy scan errors (not recommended)"